test/framework/src/main/java/org/elasticsearch/indices/analysis/AnalysisFactoryTestCase.java

```diff
@@ -447,18 +464,24 @@ public void testPreBuiltMultiTermAware() {
                 expected.add(tokenizer);
             }
         }
-        for (Map.Entry<PreBuiltTokenFilters, Class<?>> entry : PREBUILT_TOKENFILTERS.entrySet()) {
-            PreBuiltTokenFilters tokenFilter = entry.getKey();
+        Map<String, PreBuiltTokenFilterSpec> preBuiltTokenFilters = AnalysisModule.setupPreBuiltTokenFilters(singletonList(plugin));
+        for (Map.Entry<String, Class<?>> entry : getPreBuiltTokenFilters().entrySet()) {
+            String name = entry.getKey();
             Class<?> luceneFactory = entry.getValue();
             if (luceneFactory == Void.class) {
                 continue;
             }
+            if (luceneFactory == null) {
+                luceneFactory = TokenFilterFactory.lookupClass(toCamelCase(name));
```

Comment:
This is the other side of that magic `null` behavior that I wasn't super comfortable with. It is convenient though.